src/services/recurringDonationService.ts

```diff
@@ -179,7 +179,8 @@ export const createRelatedDonationsToStream = async (
       const donation = Donation.create({
         amount: normalizeNegativeAmount(
           streamPeriod.amount,
-          tokenInDb!.decimals,
+          // All Super fluid tokens have 18 decimals even USDSx https://optimistic.etherscan.io/token/0x8430f084b939208e2eded1584889c9a66b90562f
```

Comment:
just curious: was the previous code tokenInDb!.decimals not giving 18 as the value ?

Reply:
@jainkrati The decimals of `USDC` is `6` https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48

But for the superfluid tokens all decimals are `18` so we should not use the decimals that we have in our DB for the tokens